jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
 
+let orValue a x =
+  match x with
+  | None -> a
+  | Some x -> x
 
+let filter f x =
+  match x with
+  | None -> None
```

Comment:
> I've seen it in every stdlib of languages that support options.

Yet not in the OCaml `Option` module.  ^.^

But still, I'd like it in, it is a pattern I use.  :-)